CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # Change log
 
 ## master (unreleased)
+[#3513](https://github.com/bbatsov/rubocop/pull/3513): Fix false positive in `Style/TernaryParentheses` for a ternary with ranges. ([@dreyks][])
```

Comment:
See the changelog for older releases. There should be a `Bugs fixed` section here.